React-Template-master/src/components/MessageModal.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import { Modal, Box, Typography, Button, useTheme } from '@mui/material';
+import { NEUTRAL } from '../theme/palette';
+import { CorrectLogo, WarningLogo } from './Logo';
+import { useTranslation } from 'react-i18next';
+
+interface ConfirmationModalProps {
+    open: boolean;
+    setOpen: Function;
+    modalContent: {
+        title: string;
+        subtitle: string;
+        type?: string;
+        buttonText: string;
+    };
+}
+
+const style = {
+    display: 'flex',
+    alignItems: 'center',
+    flexDirection: 'column',
+    border: 'none',
+    position: 'absolute',
+    top: '50%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+    width: '408px',
+    borderRadius: '4px',
+    height: '268px',
+    boxShadow: 24,
+    p: '20px',
+    boxSizeing: 'content-box'
+};
+
+export const MessageModal = (props: ConfirmationModalProps): JSX.Element => {
```

Comment:
You can destructure the props here itself

Reply:
okay